test/controllers/presenter.controller.spec.ts

```diff
@@ -105,15 +134,16 @@ describe("PRESENTER controller", () => {
     });
 
     test(`renders the presenter page with back link URL correctly set`, async () => {
-      mockGetApplicationData.mockReturnValueOnce({ [PresenterKey]: PRESENTER_OBJECT_MOCK });
+      mockGetApplicationData.mockReturnValue({ [PresenterKey]: PRESENTER_OBJECT_MOCK });
       const resp = await request(app).get(PRESENTER_URL);
 
       expect(resp.status).toEqual(200);
       expect(resp.text).toContain(OVERSEAS_NAME_URL);
     });
 
     test("catch error when renders the presenter page", async () => {
-      mockGetApplicationData.mockImplementationOnce( () => { throw new Error(ANY_MESSAGE_ERROR); });
+      // mockGetApplicationData.mockImplementationOnce( () => { throw new Error(ANY_MESSAGE_ERROR); });
```

Comment:
comment can be removed

Reply:
yeah, i would remove them in case we never get round to doing the rest of the work!

Reply:
sure, all removed now.